### ISSUE TITLE
Unhandled exception editing blog post content item

### ISSUE THREAD

**Issue by petedavis** [OrchardCMS/OrchardCore#2027]
Repeated this twice from the latest dev branch. 

Create a new site using the Blog recipe, and then attempt to edit the blog post. The following unhandled exception is thrown:

```
An unhandled exception occurred while processing the request.
Exception: Shape type 'Markdown_Editor' not found
OrchardCore.DisplayManagement.Implementation.DefaultHtmlDisplay.ExecuteAsync(DisplayContext context) in DefaultHtmlDisplay.cs, line 128
````

**Comment by awyl** [comment]
Awaiting the fix to merge in #2022...

**Comment by petedavis** [comment]
Tested and resolved
